probe-rs/src/architecture/arm/core/exception_handling/armv7m.rs

```diff
@@ -107,11 +112,11 @@ impl Cfsr {
 
         Ok(Some(if self.bf_address_register_valid() {
             format!(
-                "BusFault ({source}) at location: {:#010x}",
+                "BusFault <Cause: {source} at location: {:#010x}>",
```

Comment:
How this is written it suggests to me that the cause was at location X rather than BusFault. Not sure I like the previous writing better, but I feel like now it's misleading :/

Reply:
I think the few people who understand how busfaults work, will understand what the location refers to.

Reply:
I think I'd like to revert these ... this is what it looks like in practice (from the test `cargo insta review` output. 

PR  version :  `function_name: "HardFault <Cause: Escalated BusFault <Cause: Precise data access error at location: 0x3ffffffc>>"`
New version: `function_name: "HardFault <Cause: Escalated BusFault <Cause: Precise data access error> at location: 0x3ffffffc>"`

@Noah, it feels like the location belongs with the `Cause`. Do you agree?

Reply:
Yeah I agree :) Sorry, I could not see the full picture!